parm/ioda/bufr2ioda/bufr2ioda_gnssro_cosmic2.json

```diff
@@ -0,0 +1,20 @@
+{
+  "data_format"      : "bufr_d",
+  "data_type"        : "gpsro",
+  "cycle_type"       : "{{ RUN }}",
+  "cycle_datetime"   : "{{ current_cycle | to_YMDH }}",
+  "dump_directory"   : "{{ DMPDIR }}",
+  "ioda_directory"   : "{{ COM_OBS }}",
+  "subsets"          : [ "NC003010" ],
+  "data_description" : "Satellite radio occultation data", 
+  "data_provider"    : "U.S. NOAA",
```

Comment:
For the data_provider, I believe we are using "U.S. NOAA NCEP". I won't comment on every single one.